### PR TITLE
added Bremen trams (BSAG)

### PR DESCRIPTION
**Pull request by lunas-bad-coding**
Bremen buses need to be added at some point, but that's too much work for me at the moment, to be honest.

### CODE REVIEW

File: line-colors.csv [hunk 1]
```diff
@@ -993,6 +993,16 @@ vbn-nwb-regio-s-bahn,RS 3,nordwestbahn,4-n1-rs3,#93bf30,#ffffff,,rectangle-round
...
+vbn-bsag,1,,8-webbtr-1,#00A54F,#ffffff,,rectangle
+vbn-bsag,1S,,8-webbtr-1S,#00A54F,#ffffff,,rectangle
+vbn-bsag,2,,8-webbtr-2,#0166B3,#ffffff,,rectangle
+vbn-bsag,3,,8-webbtr-3,#00AEEF,#ffffff,,rectangle
+vbn-bsag,4,,8-webbtr-4,#EE1D23,#ffffff,,rectangle
+vbn-bsag,5,,8-webbtr-5,#00AAB8,#ffffff,,rectangle
+vbn-bsag,5S,,8-webbtr-5S,#00AAB8,#ffffff,,rectangle
+vbn-bsag,6,,8-webbtr-6,#FECA0A,#000000,,rectangle
+vbn-bsag,8,,8-webbtr-8,#8BC63E,#000000,,rectangle
+vbn-bsag,10,,8-webbtr-10,#00AAB8,#ffffff,,rectangle
```
Comment (vainamov) [inline line-colors.csv:1005]: Hey, thank you for your contribution! Can you change the hex values to use lowercase letters?

```suggestion
vbn-bsag,1,,8-webbtr-1,#00a54f,#ffffff,,rectangle
vbn-bsag,1S,,8-webbtr-1S,#00a54f,#ffffff,,rectangle
vbn-bsag,2,,8-webbtr-2,#0166b3,#ffffff,,rectangle
vbn-bsag,3,,8-webbtr-3,#00aeef,#ffffff,,rectangle
vbn-bsag,4,,8-webbtr-4,#ee1d23,#ffffff,,rectangle
vbn-bsag,5,,8-webbtr-5,#00aab8,#ffffff,,rectangle
vbn-bsag,5S,,8-webbtr-5S,#00aab8,#ffffff,,rectangle
vbn-bsag,6,,8-webbtr-6,#feca0a,#000000,,rectangle
vbn-bsag,8,,8-webbtr-8,#8bc63e,#000000,,rectangle
vbn-bsag,10,,8-webbtr-10,#00aab8,#ffffff,,rectangle
```

Comment (lunas-bad-coding) [reply line-colors.csv:1005]: sure thing :)

File: line-colors.csv [hunk 2]
```diff
@@ -993,6 +993,16 @@ vbn-nwb-regio-s-bahn,RS 3,nordwestbahn,4-n1-rs3,#93bf30,#ffffff,,rectangle-round
 vbn-nwb-regio-s-bahn,RS 30,nordwestbahn,4-n1-rs30,#4faf3b,#ffffff,,rectangle-rounded-corner
 vbn-nwb-regio-s-bahn,RS 4,nordwestbahn,4-n1-rs4,#e2000b,#ffffff,,rectangle-rounded-corner
 vbn-nwb-regio-s-bahn,RS 6,nordwestbahn,4-n1-rs6,#899bb9,#ffffff,,rectangle-rounded-corner
+vbn-bsag,1,,8-webbtr-1,#00a54f,#ffffff,,rectangle
+vbn-bsag,1S,,8-webbtr-1S,#00a54f,#ffffff,,rectangle
```
Comment (marhei) [inline line-colors.csv:997]: HAFAS line code needs to be en all lower case!

**Review by marhei — CHANGES_REQUESTED** [review]

File: line-colors.csv (hunk 2 above)
Comment (lunas-bad-coding) [reply line-colors.csv:997]: my bad, didn't know that!!!

**Review by lunas-bad-coding — COMMENTED** [review]
changes look good to me

**Review by marhei — APPROVED** [review]

**Pull request merged**